UnicornRender/video/include/unicorn/video/Mesh.hpp

```diff
@@ -24,14 +24,10 @@ namespace video
  */
 struct Vertex
 {
-    /**
-     * @brief Position of vertex
-     */
+    /** @brief Position of vertex */
```

Comment:
`of vertex` can be omitted - you're in a struct called `Vertex` after all. Also try using `Vertex position` instead of `Position of vertex` because it just looks cleaner